### ISSUE TITLE
questions on cls_fine

### ISSUE THREAD

**Issue by feiyu12138** [GrumpyZhou/patch2pix#10]
Hi,

Thank you for your excellent work! I'd like to ask for some advice for training this model. 

When I modified this model on my own datasets, I encounter a problem that it converges to a point where the loss of cls_fine  = 0, but the metric, Specificity of cls_fine, is about  0.01, which means it has a weak ability to distinguish the false match. May I get some advice for improving the performance further?

**Comment by feiyu12138** [comment]
Hi Zhou,

Thank you for your reply! This is the end of my training log,
>Epoch:200 Skipped:0 Loss:pair=0.08 cls_mid=0.01 cls_fine=0.00 epi_mid=0.01 epi_fine=0.00 
Cls_mid:rec=1.00 prec=1.00 spec=1.00 acc=1.00 f1=1.00 
Cls_fine:rec=0.98 prec=1.00 spec=0.05 acc=0.98 f1=0.99 
Match:cmid_gt=18.17 mmid_gt=0.01 mfid_gt=0.01 ffid_gt=0.00 
Epoch training time: 25.04s
Validation setting: panc=1
Finished, time:5981.2921s

I'm training the model based on the pretrained model located at ./pretrained/patch2pix_pretrained.pth, 

the command's like: python -m train_patch2pix --gpu 1 \
    --epochs 200 --batch 2 \
    --save_step 100 --plot_counts 20 --data_root "/mnt/sdd/yeluoxin/datasets" \
    --image_path "train" --hybrid True \
    --change_stride --panc 8 --ptmax 400 \
    --pretrain 'pretrained/patch2pix_pretrained.pth' \
    -lr 0.0005 -lrd 'multistep' 0.2 5 \
    --cls_dthres 50 5 --epi_dthres 50 5 --ptmax 150 \
    -o 'output/patch2pix' --cthres 0.9

The marked point is that I wanna deploy this model on multi modal(z.B IR and RGB image).

If you'd like to provide me with some advice, please feel free to contact me(this issue or Wechat if convenient)
